src/components/views/rooms/AppsDrawer.js

```diff
@@ -85,7 +85,8 @@ module.exports = React.createClass({
             case 'appsDrawer':
                 // When opening the app draw when there aren't any apps, auto-launch the
                 // integrations manager to skip the awkward click on "Add widget"
-                if (action.show) {
+                // assuming there is a scalarClient
+                if (action.show && this.scalarClient) {
```

Comment:
This would cause utter confusion if someone had managed to dispatch an `appsDrawer` with their `scalarClient` falsey